includes/class-newspack-newsletters.php

```diff
@@ -1176,17 +1174,17 @@ public static function is_newsletter_sent( $post_id ) {
 		}
 
 		/** Detect meta that determines the sent state */
-		$sent = get_post_meta( $post_id, 'newsletter_sent', true );
-		if ( 0 < $sent ) {
+		$sent         = get_post_meta( $post_id, 'newsletter_sent', true );
+		$is_published = 'publish' === get_post_status( $post_id );
+		$publish_date = $is_published ? get_post_datetime( $post_id, 'date', 'gmt' )->getTimestamp() : 0;
```

Comment:
This only considers the legacy check, which doesn't account for private sent newsletters (which has post status set to `private`).

```suggestion
		$sent         = get_post_meta( $post_id, 'newsletter_sent', true );
		$is_published = 'publish' === get_post_status( $post_id ) || $sent;
		$publish_date = $is_published ? get_post_datetime( $post_id, 'date', 'gmt' )->getTimestamp() : 0;
```

Reply:
You're right, thanks!